docs/superpowers/plans/2026-04-30-v3.20.0-release.md

```
@@ -0,0 +1,377 @@
+
+- [ ] **Step 0.3: Baseline gate (smoke).**
+```bash
+vendor/bin/phpstan analyse --memory-limit=1G
+vendor/bin/phpcs
+vendor/bin/phpunit
+semgrep --config=.semgrep/rules.yml --error Simple-PHP-IPAM/
+```
+Expected: all green. If anything red on a clean checkout of `main`, STOP — pre-existing failure must be triaged first.
+
+- [ ] **Step 0.4: Memory MCP — open release entity.**
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)